Directory.Build.props

```diff
@@ -58,6 +57,7 @@
     <NoWarn>$(NoWarn);NU5125</NoWarn>
     <!-- Sets deterministic compilation (/deterministic compiler flag) -->
     <Deterministic>true</Deterministic>
+    <AccelerateBuildsInVisualStudio>true</AccelerateBuildsInVisualStudio>
```

Comment:
Moved this out of the 'Project Layout' section (for paths and stuff) into the 'General' section.